Source/OxyPlot/Svg/SvgRenderContext.cs

```diff
@@ -187,7 +187,7 @@ public override void DrawRectangle(OxyRect rect, OxyColor fill, OxyColor stroke,
                     for (var i = lines.Length - 1; i >= 0; i--)
                     {
                         var line = lines[i];
-                        var size = this.MeasureText(line, fontFamily, fontSize, fontWeight);
+                        _ = this.MeasureText(line, fontFamily, fontSize, fontWeight); // TODO: Is this necessary?
```

Comment:
I suppose not. There's another one on line 177.

Reply:
Will remove both, thanks

Reply:
There was another one on line 200. I removed the 3